tests/test_opener.py

```diff
@@ -11,38 +11,14 @@
 from __future__ import absolute_import, print_function
 
 from conftest import mkurl
-from fs.opener import opener
+from fs.opener import open_fs
 
-from xrootdpyfs.opener import XRootDPyOpener  # no-qa
 
-
-def test_parse(tmppath):
-    """Test parse."""
+def test_open_fs_create(tmppath):
+    """Test open with create."""
     rooturl = mkurl(tmppath)
-    fs, path = opener.parse(rooturl + "/data")
```

Comment:
Question: I'm guessing this is no longer relevant because of the API change from ``opener`` to ``open_fs``?

Reply:
Yes: the opener is not completely changed, and you only have one method `open_fs`, which returns a filesystem instance. `parse` or `opendir` have been removed.
See: https://docs.pyfilesystem.org/en/latest/extension.html#opener

I will have to change and different usage of this. In `invenio-files-rest`, this has been already fixed when upgrading to `fs 2`.